hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneManagerLock.java

```diff
@@ -600,7 +604,8 @@ public enum Resource {
     USER_LOCK((byte) 3, "USER_LOCK"), // 15
 
     S3_SECRET_LOCK((byte) 4, "S3_SECRET_LOCK"), // 31
-    PREFIX_LOCK((byte) 5, "PREFIX_LOCK"); //63
+    KEY_PATH_LOCK((byte) 5, "KEY_PATH_LOCK"), //63
```

Comment:
@tanvipenumudy please add one test case to cover the case where PREFIX_LOCK can be acquired after KEY_PATH_LOCK and reverse should throw exception.

Reply:
We are having existing UTs which cover lock precedence: `TestOzoneManagerLock.testLockingOrder()`, `TestOzoneManagerLock.testLockViolationsWithOneHigherLevelLock()`, `TestOzoneManagerLock.testLockViolations()` iterating over all `OzoneManagerLock.Resource.values()`. 

The existing UTs (`TestOzoneManagerLock.reacquireResourceLock()`) also cover re-acquiring locks over these `OzoneManagerLock.Resource.values()`